app/broadcast_message/rest.py

```diff
@@ -118,6 +119,15 @@ def update_broadcast_message_status(service_id, broadcast_message_id):
 
     broadcast_message.status = new_status
 
+    current_app.logger.info(
+        f'broadcast_message {broadcast_message_id} moving from {broadcast_message.status} to {new_status}'
+    )
     dao_update_broadcast_message(broadcast_message)
 
+    if new_status == BroadcastStatusType.BROADCASTING:
+        send_broadcast_message.apply_async(
+            kwargs={'broadcast_message_id': str(broadcast_message.id)},
+            queue=QueueNames.NOTIFY
```

Comment:
Oh was this queue just picked as a short term solution and in the medium term we will put all the CBC related tasks on to their own queue?